lizmap/modules/lizmap/classes/lizmapTiler.class.php

```diff
@@ -602,4 +599,16 @@ public static function getTileBbox($tileMatrixSet, $tileMatrixId, $tileRow, $til
 
         return (string) $minx.','.(string) $miny.','.(string) $maxx.','.(string) $maxy;
     }
+
+    private static function normalizeProjectScales($project)
+    {
+        $scales = array_merge(array(), $project->getOption('mapScales'));
+        if (reset($scales) == 0) {
```

Comment:
If we are sure that the array contains values ordered from minimum value to maximum values, why not just use 
```suggestion
        if ($scales[0] == 0) {
```

Reply:
We're sure :  
https://github.com/3liz/lizmap-plugin/blob/392299aa54875c4e409ef5b7e31f39371d4f6292/lizmap/plugin.py#L1371